nemo/collections/nlp/models/language_modeling/megatron_gpt_peft_models.py

```diff
@@ -17,6 +17,12 @@
 from pytorch_lightning.trainer.trainer import Trainer
 
 from nemo.collections.nlp.models.language_modeling.megatron_gpt_sft_model import MegatronGPTSFTModel
+from nemo.collections.nlp.modules.common.megatron.adapters.mcore_mixins import (
```

Comment:
@arendu you might wanna review these changes

Reply:
those are from #7276 which are already reviewed by @arendu

Reply:
awesome